docs/building-images.md

```diff
@@ -157,7 +157,46 @@ operatingSystem:
   * `additionalRepos` - Optional; List of third-party RPM repositories that will be added to the package manager of the OS.
   * `sccRegistrationCode` - Optional; SUSE Customer Center registration code, used to connect to SUSE's internal RPM repositories.
 
-## SUSE Manager (SUMA)
+### Kubernetes
+
+The Kubernetes configuration section is another entirely optional one.
+It contains all necessary settings to configure and bootstrap a Kubernetes cluster.
+
+```yaml
+kubernetes:
+  version: v1.29.0+rke2r1
```

Comment:
Can we make this 1.28 in the docs, at least this time around? Reason being, our first release will be based on 1.28, and let's make sure that 1.29 doesn't get accidentally deployed ;-)